packages/websocket-plugin/src/websocket-handler.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable, Inject } from '@angular/core';
 import { WebSocketSubject } from './websocket-subject';
-import { Actions, Store, getValue, ofActionDispatched } from '@ngxs/store';
+import { Store, getValue, ofActionDispatched } from '@ngxs/store';
+import { InternalActions } from '@ngxs/store/src/actions-stream';
```

Comment:
@amcdnl This import is a bit strange because it is accessing an internal class. 
I did not want to expose the InternalActions as part of the public api.
As far as I can see, this seems to work. the packager ran fine and the tests all passed.
I don't have a sample websockets project to test with. Could you test one once merged?

Reply:
Why do we need to use internal actions here? Seems like the normal actions would be fine.

Reply:
Because the other changes in this PR makes normal actions run back in the angular zone and I wouldn't want to assume that the websocket stuff should be run in the angular zone again.

Reply:
I think for these small things it would be fine.

Reply:
Ok. I'll revert the changes to the websockets part